core/src/nucleus/ribosome/api/commit.rs

```diff
@@ -27,20 +23,20 @@ fn build_validation_data_commit(
     // TODO: populate validation data with with chain content
     // I have left this out because filling the valiation data with
     // chain headers and entries does not work as long as ValidationData
-    // is defined with the type copies i've put in wasm_utils/src/validation.rs.
+    // is defined with the type copies i've put in wasm_utils/src/api_serialization/validation.rs.
```

Comment:
Avoid the word "i" and "I've" since the reader may not have any context as to who

Reply:
`git blame` gives that context :)

Reply:
I think you might feel differently if someone else started scattering the word "I" around the code comments. It's just an extra step, and not everyone knows how to use git blame, and those people could still be reading the source code. I really differ here

Reply:
Using "I" in code comments creates an informal tone that IMO conveys that this meant by a developer for a developer. It is not meant as a code comment to stay there or to be used in docs. Hence the // instead of /// and all of it framed under TODO. I deliberately used this style here in that way.

Also, it is not changed in this PR. But I will take this comment block out within the coming few days when I'll work on the todo.